capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -224,6 +224,19 @@ For example if you need to show animated illustrations you can use a `Lottie
 
 You can find more details `here <customization-guide.html>`_.
 
+Camera
+~~~~~
+
+New Features
+++++++++++++
+
+We implemented image cropping. Parts of the image that appears outside the white camera frame will be cut out from the final image.
```

Comment:
[Request] Here we should also mention the bottom navigation bar and activity indicator customisation. In the same way we did it for Help and Analysis.

Reply:
Hey, @AdnanYupi, please also add info bout the activity indicator which is shown during image processing (cropping).